src/renderers/shared/fiber/__tests__/ReactIncrementalScheduling-test.js

```diff
@@ -340,4 +340,13 @@ describe('ReactIncrementalScheduling', () => {
     // animation priority.
     expect(ReactNoop.getChildren()).toEqual([span(1)]);
   });
+
+  it('can force synchronous updates with syncUpdates, even inside batchedUpdates', () => {
+    ReactNoop.batchedUpdates(() => {
+      ReactNoop.syncUpdates(() => {
+        ReactNoop.render(<span />);
+        expect(ReactNoop.getChildren()).toEqual([span()]);
```

Comment:
You could guard against this with:
```jsx
it('can force synchronous updates with syncUpdates, even inside batchedUpdates', (done) => {
  ReactNoop.batchedUpdates(() => {
    ReactNoop.syncUpdates(() => {
      ReactNoop.render(<span />);
      expect(ReactNoop.getChildren()).toEqual([span()]);
      done();
    });
  });
});
```

But yeah I agree, probably not necessary. 😄